webapp/sources/rudder/rudder-web/src/main/resources/rudder-system-global-parameter.conf

```diff
@@ -0,0 +1,10 @@
+[{
+  "name": "rudder"
+  "description": "This parameter defines important properties for Rudder and should not be modified. If you modify them, they will be resetted at next restart."
```

Comment:
I think the descprition should explain how to override it. 

This property can be overriden by defining property rudder on groups / nodes

Reply:
But it will be true for all properties. It should be in the global documentation, not here. OK since it's the first.